### PR TITLE
Fixed issues found during code review.

### PR DESCRIPTION
**Pull request by Remki**

### CODE REVIEW

**Comment by segiddins** [comment]
This largely looks awesome!

File: SSDynamicText/SSDynamicButton.h [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 #import <UIKit/UIKit.h>
 
-/*
+/**
  While creating this button in xib, don't forget to change it type to Custom !!
```
Comment (Grubas7) [inline SSDynamicText/SSDynamicButton.h]: In other comments there are an asters in every single line.

File: SSDynamicText/SSDynamicAttributedTextSizable.h [hunk 2]
```diff
@@ -10,12 +10,16 @@
...
- Framework is using it under the hood sometimes after layouts or even setText calls it. Because of that we cannot override
- default attributeText setter to change font, sometimes it change font at random.
+/**
+ * TextView and TextField sometimes calls setAttributedText even when we work with normal Text.
+ * Framework is using it under the hood sometimes after layouts or even setText calls it. Because of that we cannot override
+ * default attributeText setter to change font, sometimes it change font at random.
+ *
+ * This is used to set attributedText which will be dynamicaly changed with font size changes.
+ * Updating this will change attributedText to dynamicAttributedText + font sizes changed with delta.
+ * @return originaly dynamicAttributedText value. To check current attributedText font sizes use @property attributedText
```
Comment (Grubas7) [inline SSDynamicText/SSDynamicAttributedTextSizable.h]: `originaly` -> `original` ?

File: SSDynamicText/NSAttributedString+SSTextSize.h [hunk 3]
```diff
@@ -10,6 +10,13 @@
 
 @interface NSAttributedString (SSTextSize)
 
+/**
+ * Create NSAttributedString with each font size changed by delta
```
Comment (Grubas7) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: `Create` -> `Creates` ?

File: SSDynamicText/NSAttributedString+SSTextSize.h [hunk 4]
```diff
@@ -10,6 +10,13 @@
...
+ * @param delta The size by which you want to change the font.
+ * A negative number will lower font.
```
Comment (Grubas7) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: `lower font` -> `decrease font size`

File: SSDynamicText/NSAttributedString+SSTextSize.h [hunk 5]
```diff
@@ -10,6 +10,13 @@
...
+ * A positive number will increase font.
```
Comment (Grubas7) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: `increase font` -> `increase font size`

File: SSDynamicText/NSAttributedString+SSTextSize.h [hunk 6]
```diff
@@ -10,6 +10,13 @@
...
+ * @return new NSAttributedString object which font size changed by delta.
```
Comment (Grubas7) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: `new NSAttributedString object which font size is changed by delta`
or
`new NSAttributedString object with font size changed by delta`

**Comment by Grubas7** [comment]
👍

File: SSDynamicText/SSDynamicButton.h (hunk 1 above)
Comment (Remki) [inline SSDynamicText/SSDynamicButton.h]: Fixed.

File: SSDynamicText/SSDynamicAttributedTextSizable.h (hunk 2 above)
Comment (Remki) [inline SSDynamicText/SSDynamicAttributedTextSizable.h]: Fixed.

File: SSDynamicText/NSAttributedString+SSTextSize.h (hunk 3 above)
Comment (Remki) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: Fixed.

File: SSDynamicText/NSAttributedString+SSTextSize.h (hunk 4 above)
Comment (Remki) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: Fixed.

File: SSDynamicText/NSAttributedString+SSTextSize.h (hunk 5 above)
Comment (Remki) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: Fixed.

File: SSDynamicText/NSAttributedString+SSTextSize.h (hunk 6 above)
Comment (Remki) [inline SSDynamicText/NSAttributedString+SSTextSize.h]: Fixed.

**Pull request closed without merging**